Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -3068,21 +3067,7 @@ void drawNormalBond(MolDraw2D &d2d, const Bond &bond, bool highlight_bond,
     d2d.drawOptions().scaleBondWidth = orig_slw;
   } else if (Bond::SINGLE == bt && (Bond::BEGINWEDGE == bond.getBondDir() ||
                                     Bond::BEGINDASH == bond.getBondDir())) {
-    // swap the direction if at1 has does not have stereochem set
-    // or if at2 does have stereochem set and the bond starts there
-    auto at1 = bond.getBeginAtom();
-    auto at2 = bond.getEndAtom();
     auto inverted = false;
```

Comment:
Do we need inverted any more?  It looks like it's always passed in false, and I don't see drawWedgedBonds being used anywhere else.
Either way, I suggest you leave it there for now, and I'll take it out of the re-factored code.  This is all in DrawMol now, so I will have to merge this by hand into that in any case.  Doing the minimum work in MolDraw2D at this point makes sense, but I could do with knowing the answer to the question.

Reply:
I don't think we need inverted anymore. I thought about taking it out here, but since I knew you were doing larger-scale changes I thought it made sense to keep things here minimal.

Reply:
Great thanks.  As soon as I have the latest mole back in its hole, I'll take it out of my code.